packages/app/src/routes/AppRoutes.tsx

```diff
@@ -10,12 +9,6 @@ import { ExtensionsPage } from '../components/Extensions/ExtensionsPage'
 import extensionsService from '../services/extensions.service'
 import { ExtensionsLayout } from '../layouts/ExtensionsLayout'
 
-const AppRoutesPageSubscription = () => {
```

Comment:
Looks like this is being used a few lines later on line 16

Reply:
Good find

Reply:
Do I need to do anything else to approve your changes?